plugin/src/main/kotlin/org/gradleweaver/plugins/jlink/JLinkPlugin.kt

```diff
@@ -66,11 +66,13 @@ open class JLinkPlugin : Plugin<Project> {
             group = JLINK_TASK_GROUP
             description = "Generates a .zip archive file of a native Java runtime image for '${options.name}'."
             from(project.tasks.getByName(jlinkTaskName).outputs)
+            archiveName = "${project.name}-${options.name.toLowerCase().replace(' ', '-')}.zip"
         }
         project.tasks.register(generateJLinkArchiveTaskName("Tar", options.name), Tar::class.java) {
             group = JLINK_TASK_GROUP
             description = "Generates a .tar.gz archive file of a native Java runtime image for '${options.name}'."
             from(project.tasks.getByName(jlinkTaskName).outputs)
+            archiveName = "${project.name}-${options.name.toLowerCase().replace(' ', '-')}.tar.gz"
             compression = Compression.GZIP
```

Comment:
This shouldn't default to GZIP.

Reply:
Setting the compression to GZIP keeps the final archive size in line with the ZIP artifact. Not specifying a compression level results in an uncompressed archive file

Reply:
It's not the responsibility of the plugin to configure this.
If the user of the plugin want's to use GZIP, they can use this:
```
tasks.withType<Tar>().configureEach {
    // Always Gzip the tar
    compression = Compression.GZIP
}
```

That's on the plugin user though, not the plugin. It's the same reason that GZIP isn't the default for the `Tar` task for the rest of Gradle.